sentinel5dl/__init__.py

```diff
@@ -89,17 +90,41 @@ def __http_request(path, filename=None):
     logger.debug(f'Requesting {url}')
     with open(filename, 'wb') if filename else io.BytesIO() as f:
         curl = pycurl.Curl()
+        if ca_info:
```

Comment:
Since certifi has been merged since can you merge `master` into this or rebase this onto `master`? There shouldn't be any certifi related changes in here.

Reply:
this line slipped in due to a merging error. didn't see that coming. the original certifi line is in 102